sphere_tesselate.f90

```diff
@@ -915,7 +915,7 @@ function find_all_triangles_surrounding_point(p,t) result(local_tess)
 ! output local_tess is tesselation containing all triangles connected to point p
    type(point), intent(in) :: p
    type(tesselation), intent(in) :: t
-   type(tesselation), intent(out) :: local_tess
+   type(tesselation)             :: local_tess
```

Comment:
If you really want to (to save me doing it), keeping the new code in the same style as the old is always handy for pedants like me.  Note that the current codebase doesn't align the variable names.